dirkpetersen/froster
# Develop (#10)

* OHSU-32 First batch of "froster config" tests

* OHSU-32 Separate requirements from setup.py and install them prior executing tests

* Added return True False logic in froster config and added extra check in froster config --aws for region

* OHSU-32 Make the install script install from current repo and branch

* Fix runnign unittests

* Set correct env variables in GA

* Fix BRANCH env in GA workflow

* error while installing froster in ga

* restore install to its original state

* Adding secrets in environment

* ignore resource warnings

* close boto3 sessions if they are not used

* Increased version number to 0.10.5

* version read from setup.py and aws sessions closed at the end

* Install froster in froster-test-config tests

* Delete unneded variables in froster-ubuntu-install test

* new version retrival implementation

* mini bugfix

* Ignore resource warning due to boto3 library

* trying to ignore resource warnings

* closing connections in AWS BOTO 3 when exiting

* Close check credentials session

* trying to ignore resource warnings in ga

* trying to delete resource warning from ga

---------
